less/components/sidebar.less

```diff
@@ -28,6 +28,17 @@ html.no-js .sidebar {
   &.teaching-materials .sidebar {
     .sidebarColorizer(@blueSidebarColor);
   }
+  &.web-literacy .sidebar {
+    .sidebarColorizer(@darkBlueSidebarColor);
+  }
+}
+
+.teach-like-mozilla, .web-literacy {
```

Comment:
I tried to find a better way to turn this one, but because .active is auto put on the anchors, and not the parent, it makes for gross parent/sibling less selectors...